onnxruntime/core/framework/session_state.cc

```diff
@@ -1185,13 +1185,12 @@ static Status VerifyEachNodeIsAssignedToAnEp(const Graph& graph, const logging::
   if (is_verbose_mode) {
     LOGS(logger, VERBOSE) << "Node placements";
     if (node_placements.size() == 1) {
-      LOGS(logger, VERBOSE) << "All nodes have been placed on [" << node_placements.begin()->first << "].";
+      LOGS(logger, VERBOSE) << " All nodes have been placed on [" << node_placements.begin()->first << "].";
     } else {
       for (const auto& [provider, node_strs] : node_placements) {
-        std::ostringstream all_nodes_str;
-        std::copy(node_strs.begin(), node_strs.end(), std::ostream_iterator<std::string>(all_nodes_str, ", "));
-        LOGS(logger, VERBOSE) << " Provider: [" << provider << "]"
-                              << ": [" << all_nodes_str.str() << "]";
+        for (const auto& node_str : node_strs) {
+          LOGS(logger, VERBOSE) << " Node [" << node_str << "] has been placed on [" << provider << "].";
```

Comment:
Any reason to repeat the provider for every node given that all nodes in nodes_strs are placed on the same provider?

Reply:
Yeah - that seems a little too verbose vs something like `CPUExecutionProvider: node_str1, nodestr_2` or

```
CPUExecutionProvider: 
node_str1
nodestr_2
```

Reply:
reasoning was to be able to search by line (e.g., with grep) and get the needed info